task_processing/util/base62.py

```diff
@@ -0,0 +1,30 @@
+import os
```

Comment:
they are very verbose. if we have task id that contains more than single uuid (which isn't impossible) logs will be unreadable. keep in mind, the data that's base62-encoded is identical to uuid bytes so we're not compromising anything here.